conf/application.conf

```diff
@@ -96,7 +96,8 @@ features {
     "bereavement",
     "crime",
     "fireOrFlood",
```

Comment:
should this be fireandflood now?

Reply:
Yes, good spot 👍

Reply:
Fixed on [d76dde7](https://github.com/hmrc/income-tax-penalties-appeals-frontend/pull/42/commits/d76dde737136a2e1e43ce027f60683fa06e63a0f)